### ISSUE TITLE
Preview support?

### ISSUE THREAD

**Issue by jleclanche** [Stephanvs/vscode-graphviz#1]
Being able to live-preview dot files would be super nice. :)

**Comment by Stephanvs** [comment]
Yeah, that'd be very cool to have. When I can find some time I'll look into it.

**Comment by Stephanvs** [comment]
Actually am pretty far in the `preview` branch, but having issues with sizing the svg element within the @code preview window, any help is greatly appreciated!

**Comment by mishak87** [comment]
Markdown Preview Enhanced have implemented it but I would rather have it in this extension
see https://shd101wyy.github.io/markdown-preview-enhanced/#/diagrams?id=plantuml
